content/docs/dvclive/api-reference/live/set_step.md

```diff
@@ -1,6 +1,6 @@
 # dvclive.set_step()
 
-Signals that the current step has ended and sets `step` to the given value.
+Signals that the current step has ended and sets step to the given value.
```

Comment:
💅🏼  The code block made sense here IMO. With plain "sets step" it's unclear what we're talking about.